### PR TITLE
Added SourceClear TravisCI configuration to run once per build

### PR DESCRIPTION
**Pull request by ooola**

### CODE REVIEW

**Comment by ooola** [comment]
This change enables the SourceClear scanner to run in a separate build. Note that it may duplicate whatever is done in the standard openjdk8 build. 

This is the most straightforward way I found to enable SourceClear to run only once per pull request.

File: sourceclear.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ "$RUN_SOURCECLEAR" = "true" ]; then
+    echo "RUN_SOURCECLEAR is set, running sourceclear"
+    curl -sSL https://download.sourceclear.com/ci.sh |  bash
+else
+    echo "RUN_SOURCECLEAR it not set, skipping"
```
Comment (aliabbasrizvi) [inline sourceclear.sh]: nit. is and not it

File: sourceclear.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```
Comment (aliabbasrizvi) [inline sourceclear.sh]: Not a fan of top level sourceclear.sh file. Please create a scripts folder and throw it in there.

Comment (wangjoshuah) [reply sourceclear.sh]: +1. Same model is used in other SDKs see https://github.com/optimizely/objective-c-sdk/tree/master/Scripts

**Comment by ooola** [comment]
This was just a PoC. Closing for now.

**Pull request closed without merging**